src/main/java/com/michalsydoryk/app/gameengine/GameConfigurator.java

```diff
@@ -0,0 +1,93 @@
+package com.michalsydoryk.app.gameengine;
+
+import com.michalsydoryk.app.board.Board;
+import com.michalsydoryk.app.board.Board2D;
+import com.michalsydoryk.app.player.Player;
+import com.michalsydoryk.app.ui.UI;
+
+public class GameConfigurator {
+    private int boardSize;
+    private int combinationSize;
+
+    GameConfiguration gameConfiguration;
```

Comment:
Composition over inheritance? Not an overkill in this case?

Reply:
There is GameBuilder which takes gameConfiguration as argument to build Game
GameConfigurator is a class which assembly gameConfiguration. 
2 ints in 9 and 10 lines are juts a variable which improve readability (I think so).
In this case inheritence is less safer.

Reply:
So it's used somewhere else - ok:)